### ISSUE TITLE
Move to album even when not owned by ourselves

### ISSUE THREAD

**Issue by SkypLabs** [ente-io/photos-app#1658]
**Describe the feature**

As of now, it is not possible to use the `Move to album` action with a shared album not owned by ourselves.

I would like to be able to move items to a shared album that I do not own. Once moved to the shared album, my items should be automatically marked as uncategorised on my side.

**Comment by vishnukvmd** [comment]
Closing this in favor of the parent issue documented @ https://github.com/ente-io/photos-app/issues/1232, fixing which will solve this use case as well 🙂

**Comment by SkypLabs** [comment]
I don't think #1232 would solve my issue given my workflow.

Every picture and video I take with my phone is uploaded to my `Camera` album. At this point, I don't know yet if the photo/video I just took will be moved to an album I own or not.

**Comment by vishnukvmd** [comment]
Hey, once #1232 is fixed, when you move an item from `Camera` to `Shared album`, it'll automatically get de-linked from `Camera`, and get moved to `Uncategorized`.

I'll reopen this issue, just to make sure that we close this after this use case has been served.

**Comment by SkypLabs** [comment]
> Hey, once https://github.com/ente-io/photos-app/issues/1232 is fixed, when you move an item from `Camera` to `Shared album`, it'll automatically get de-linked from Camera, and get moved to `Uncategorized`.

So if I understand correctly, once #1232 is done, it will be possible to use the `Move to album` action with any album (both owned by myself and shared with me)? If yes, that would indeed fix my issue.

**Comment by vishnukvmd** [comment]
Yes, that will enable this behavior. But I'll keep this open since there will have to be minor changes to the UX to add shared albums to the possible destinations of a "Move".
